packages/db/prisma/schema.prisma

```diff
@@ -144,11 +146,14 @@ model AilaUserModification {
   createdAt DateTime @default(now()) @map("created_at")
   updatedAt DateTime @updatedAt @map("updated_at")
 
-  userId     String                     @map("user_id")
-  chatId     String                     @map("chat_id")
-  messageId  String                     @map("message_id")
-  textForMod String                     @map("text_for_mod")
-  action     AilaUserModificationAction @map("action")
+  userId          String                     @map("user_id")
+  chatId          String                     @map("chat_id")
+  messageId       String                     @map("message_id")
+  textForMod      String?                    @map("text_for_mod")
```

Comment:
Made this nullable, will remove in a subsequent migration, as it's not storing meaningful data

Reply:
why not just remove now?